discussion.md

```diff
@@ -201,3 +201,145 @@ Even though a logo will be stored in a binary format such as `jpg` or `png`,
 you can expect it will remain fairly static through the lifetime of your repository.
 On the rare occasion that branding does change,
 you will probably just want to replace the logo completely rather than merge little differences in.
+
+##Removing a file
+Adding and modifying files are not the only actions one might take
+when working on a project.  It might be required to remove a file
+from the repository.
+
+Create a new file for the planet Nibiru:
+
+~~~{.bash}
+$ echo "This is another name for fake planet X" > nibiru.txt
+~~~
+
+Now add to the repository like you have learned earlier:
+
+~~~{.bash}
+$ git add nibiru.txt
+$ git commit -m 'adding info on nibiru'
+$ git status
+~~~
+~~~{.output}
+On branch master
+nothing to commit, working directory clean
+~~~
+
+Nibiru is not a real planet.  That was a silly idea.  Let us remove
+it from the disk and then from the repository:
+
+~~~{.bash}
```

Comment:
You don't need to do this. `git rm` removes the file from the working tree  and the index [1], meaning that it does a `rm nibiru.txt` when you do a `git rm nibiru.txt` 

[1] http://git-scm.com/docs/git-rm